app/models/distribution.rb

```diff
@@ -54,6 +57,11 @@ def total_quantity
 
   private
 
+  def initialize_issued_at
+    self.issued_at ||= self.created_at
+    save
```

Comment:
Does this hit the database twice for every initial distribution creation since it's a callback for `after_create`? But I guess it would have to because the `created_at` field isn't populated until the record is created. 

Would it be simpler to just set this field to the current Date/Time? Or does this field have to be exactly the same value as `created_at`?